Mathlib/Data/Matrix/Rank.lean

```diff
@@ -101,6 +102,26 @@ theorem rank_of_isUnit [StrongRankCondition R] [DecidableEq n] (A : Matrix n n R
   exact rank_unit A
 #align matrix.rank_of_is_unit Matrix.rank_of_isUnit
 
+/-- Right multiplying by an invertible matrix does not change the rank -/
+lemma rank_mul_eq_left_of_isUnit_det [DecidableEq n]
+    (A : Matrix n n R) (B : Matrix m n R) (hA : IsUnit A.det) :
```

Comment:
No action needed; but the lemma above is called `of_is_unit` and takes `IsUnit A` instead of `IsUnit A.det`.
Deciding between these is probably a job for wider cleanup.